starboard/shared/starboard/media/codec_util_test.cc

```diff
@@ -267,6 +267,18 @@ TEST(CodecUtilTest, DoesNotParse1AsPcmForNonWavSubtypes) {
   EXPECT_EQ(GetAudioCodecFromString("1", "webm"), kSbMediaAudioCodecNone);
 }
 
+TEST(CodecUtilTest, ParsesIamfCodec) {
+  EXPECT_EQ(GetAudioCodecFromString("iamf", ""), kSbMediaAudioCodecIamf);
+  EXPECT_EQ(GetAudioCodecFromString("iamf.000.000.Opus", ""),
```

Comment:
Consider adding checks for other profiles, e.g. iamf.001.255.Opus.  Maybe with some invalid values, e.g. iamf.000.256.Opus, or iamf.000.000.invalid.

Reply:
Discussed offline. In this PR, IAMF codec params are only lightly parsed, so the invalid values suggested will still pass. Though since all other codecs params are checked for exact values ([except AAC](https://github.com/youtube/cobalt/blob/387b85ab57ca09ba89ed41a3a4fb21461bc0ab0c/starboard/shared/starboard/media/codec_util.cc#L82)),I think we can check for validity in `GetAudioCodecFromString()`.

In that case I'll leave this as is in this PR, and update `GetAudioCodecFromString()` and the test in a follow up https://github.com/youtube/cobalt/pull/4061

Reply:
Consider moving this comment as commit message and/or comment here.

Reply:
Updated commit message.